dev-libs/vectorscan/vectorscan-5.4.11.ebuild

```diff
@@ -0,0 +1,74 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake edo flag-o-matic
+
+DESCRIPTION="A portable fork of the high-performance regular expression matching library"
+HOMEPAGE="
+	https://www.vectorcamp.gr/vectorscan/
+	https://github.com/VectorCamp/vectorscan
+"
+SRC_URI="
+	https://github.com/VectorCamp/vectorscan/archive/refs/tags/${PN}/${PV}.tar.gz
+		-> ${P}.tar.gz
+"
+
+S="${WORKDIR}/${PN}-${P}"
+
+LICENSE="BSD"
+SLOT="0/5"
+KEYWORDS="~amd64"
+IUSE="cpu_flags_x86_avx2 cpu_flags_x86_sse4_2"
+
+DEPEND="
+	dev-libs/boost:=
+"
+RDEPEND="
+	${DEPEND}
+	!dev-libs/hyperscan
+"
+BDEPEND="
+	dev-util/ragel
+	virtual/pkgconfig
+"
+
+REQUIRED_USE="
+	x86? ( cpu_flags_x86_sse4_2 )
+	amd64? ( cpu_flags_x86_sse4_2 )
+"
+
+src_prepare() {
+	local sedargs=(
+		# Respect user -m flags (march/mtune)
+		-e '/set(ARCH_CX*_FLAG/d'
+		# Respect user -O flags
+		-e '/set(OPT_CX*_FLAG/d'
+	)
+	sed -i "${sedargs[@]}" CMakeLists.txt cmake/cflags-x86.cmake || die
+	cmake_src_prepare
+}
+
+src_configure() {
+	use cpu_flags_x86_avx2   && append-flags -mavx2
+	use cpu_flags_x86_sse4_2 && append-flags -msse4.2
```

Comment:
I don't mind this, but we do have a QA rule saying you shouldn't attempt to align text.
https://projects.gentoo.org/qa/policy-guide/ebuild-format.html#pg0101

Reply:
The policy is related to tabulators, which is not this case. This will be always aligned independently on different tab widths.

Reply:
> This will be always aligned independently on different tab widths.

You can take it literally. Proof of example :)
https://dev.gentoo.org/~juippis/pics/Screenshot_20231127_204757.png

Reply:
ok :) I will change it.